test/module/irohad/ametsuchi/flat_file_test.cpp

```diff
@@ -60,8 +63,9 @@ TEST_F(BlStore_Test, BlockStoreWhenRemoveBlock) {
   log_->info("----------| Simulate removal of the block |----------");
   // Remove file in the middle of the block store
   {
-    log_->info("----------| create blockstore and insert 3 elements "
-                   "|----------");
+    log_->info(
+        "----------| create blockstore and insert 3 elements "
```

Comment:
is it really necessary to write these decorators?

e.g. `----------| something |----------`

Reply:
It was initially written like this

Reply:
If you are reworking logging string better to remove them and write `given`, `when`, `then` notation.

Reply:
I just ran CLang Format :D 
I did not change anything